Fix parallel build issue

Martin Schlemmer <azarah@gentoo.org> (23 May 2006)

--- Linux-PAM-0.99.3.0/doc/specs/Makefile.am	2006-05-23 14:31:07.000000000 +0200
+++ Linux-PAM-0.99.3.0.az/doc/specs/Makefile.am	2006-05-23 14:31:30.000000000 +0200
@@ -9,7 +9,7 @@ EXTRA_DIST = draft-morgan-pam.raw std-ag
 all: padout spec
 	test -f rfc86.0.txt || cp -p $(srcdir)/rfc86.0.txt .
 
-spec: draft-morgan-pam.raw
+spec: padout draft-morgan-pam.raw
 	./padout < $(srcdir)/draft-morgan-pam.raw > draft-morgan-pam-current.txt
 
 noinst_PROGRAMS = padout
